lib/libavc.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2015 Google.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""An encoder using the libavc encoder from Android M.
+
+"""
+import encoder
+import file_codec
+
+class LibavcCodec(file_codec.FileCodec):
+
+  def __init__(self,
+               name='libavc',
+               formatter=None):
+    self.name = name
+    self.codecname = 'libavc'
+    super(LibavcCodec, self).__init__(
+      name,
```

Comment:
Suspect that it was a result of different Emacs setups. But indent-by-4 seems more sensible.